### ISSUE TITLE
Problem of learning rate

### ISSUE THREAD

**Issue by qzp2018** [yanmin-wu/EDA#5]
Thanks for your excellent job!When doing experiments,I have find the model is particularly sensitive to learning rates.When training with one 3090GPU,without adjusting lr,the results are as follows:
The long line below is the baseline,the other short lines are the results training on single GPU without adjusting lr
![113573aee00e537d09002d5b23e169e](https://user-images.githubusercontent.com/41356564/228110448-a80d7b34-442e-4831-bd1a-3c991d47dbad.png)
![9f5315541308ee4fd5e3e26c3fdabb3](https://user-images.githubusercontent.com/41356564/228111177-ef06960f-f072-42eb-8c33-0cb84c17f77d.png)
Even when training with two 3090GPUs,adjusting learing rate such as multiple 2 or multiple 1.4 will show the same phenomenon.
So do you have any advice for this problem?Thank you very much!

**Comment by yanmin-wu** [comment]
We also observe this phenomenon and believe it is related to the method of updating gradients with DistributedDataParallel during parallel training. When the number of GPUs decreases, the total sample size decreases, and the learning rate used for training with 4 GPUs may be too high.
